docs/configuration.rst

```diff
@@ -70,6 +70,12 @@ below:
      - True
      - Enables sending :ref:`telemetry <Instrumentation Telemetry>` events to the agent.
 
+       .. _dd-telemetry-heartbeat-rate:
```

Comment:
Does this really need to be this public right now? Why would a user need to configure this setting?

I'd prefer we hold off until there is a proven need on the user side.